src/test/java/seedu/duke/component/CapacitorTest.java

```diff
@@ -0,0 +1,30 @@
+package seedu.duke.component;
+
+import org.junit.jupiter.api.Test;
+import seedu.duke.DukeException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class CapacitorTest {
```

Comment:
Perhaps the `addSeries()` and `addParallel()` methods could be tested as well? They are specific to the `LoadComponent` subclasses.

Reply:
Should I remove the previously created LoadComponentTest class and split it into the respective component test subclasses?

Reply:
Oh that's a good point. No need to put them here then, you can just add test cases to `LoadComponentTest`?

Reply:
Yes, that's a good idea